test/functional/rpc_generateblock.py

```diff
@@ -16,15 +17,14 @@ class GenerateBlockTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
 
-    def skip_test_if_missing_module(self):
-        self.skip_if_no_wallet()
-
     def run_test(self):
         node = self.nodes[0]
+        miniwallet = MiniWallet(node)
+        miniwallet.rescan_utxos()
 
         self.log.info('Generate an empty block to address')
-        address = node.getnewaddress()
-        hash = self.generateblock(node, output=address, transactions=[])['hash']
+        address = miniwallet.get_address()
+        hash = self.generateblock(node, address, [])['hash']
```

Comment:
why are the named args removed? Again, why is this line changed at all?

Reply:
The args were not named on the other calls so I removed it here for consistency too. I'll add it back if that it's better to leave it as it.